R/seeding.R

```diff
@@ -0,0 +1,48 @@
+##' Seeding function for use with the particle filter
+##' 
+##' @title Compare model to ICU data
+##' 
+##' @param model An \code{odin_model} object
+##' 
+##' @param data The data, must be constructed with
+##'   \code{particle_filter_data}
+##'
+##' @param pars_seeding Parameters for seeding
+##' 
+##' @importFrom stats rpois
+##' 
+##' @export
+seeding_function <- function(model,
+                             data,
+                             pars_seeding = NULL
```

Comment:
It would be useful to demonstrate the format of `pars_seeding` somewhere. We want the default to be `NULL` so perhaps in the docstring of this function is best

Reply:
Done